README.rst

```diff
@@ -36,19 +36,18 @@ This also supports ordinal number conversion (for English only).
 >>> parse("First day of year two thousand")
 '1 day of year 2000'
 
+Parsing a number
+---------------------------
```

Comment:
```suggestion
Parsing a number
----------------
```

Looks slightly better, and otherwise Sphinx logs a warning, I believe.

Reply:
Thanks for pointing it out, I have corrected it.:smile: